mempool/exports.sh

```diff
@@ -4,4 +4,14 @@ export APP_MEMPOOL_API_IP="10.21.21.27"
 export APP_MEMPOOL_DB_IP="10.21.21.28"
 
 hidden_service_file="${EXPORTS_TOR_DATA_DIR}/app-${EXPORTS_APP_ID}/hostname"
-export APP_MEMPOOL_HIDDEN_SERVICE="$(cat "${hidden_service_file}" 2>/dev/null || echo "")"
\ No newline at end of file
+export APP_MEMPOOL_HIDDEN_SERVICE="$(cat "${hidden_service_file}" 2>/dev/null || echo "")"
+
+# Check if Lightning Node app is installed and export required variables if so
+# The Lightning Node app is optional and not listed in the `required` field of the umbrel-app.yml file, so we need to do this for compatibility with umbrelOS >=1.3 where only exports of required apps are sourced.
+installed_apps=$("${UMBREL_ROOT}/scripts/app" ls-installed)
```

Comment:
@nmfretz I just pulled the merged changes and I still don't see the exported variables.

Trying to debug, when I run `umbrel/scripts/app ls-installed` I get `This script requires "docker-compose" to be installed`, so it looks like the check below is failing.

Were you able to test this locally?

Reply:
Hey, yes I tested this in production on both arm64 and x86.

You will not be able to successfully run `umbrel/scripts/app ls-installed` manually in umbrelOS 1.0+. This is a legacy umbrelOS 0.5 script; however, `umbreld` in umbrelOS 1.0+ will properly handle this when sourcing the `exports.sh` file and it will translate it to the proper function call.

> I just pulled the merged changes and I still don't see the exported variables.

I can help troubleshoot. How did you end up updating the mempool app to this current version (`3.0.1-hotfix`)?

Reply:
@nmfretz I usually do the following:

1) Clone `umbrel-apps` to the home dir (often from our clone, let's call it `mempool-umbrel-apps`, where we make the changes to the app)
2) Remove the local app definition`rm -rf umbrel/app-stores/getumbrel-umbrel-apps-github-53f74447/mempool/`
3) Copy the app from the cloned repo `cp -R ~/mempool-umbrel-apps/mempool ~/umbrel/app-stores/getumbrel-umbrel-apps-github-53f74447/mempool`
4) Update the app `umbreld client apps.restart.mutate --appId mempool`

Then I checked the running container:

1) `sudo su`
2) `docker ps | grep backend`
3) `docker exec -it <container_id> /bin/bash`
4) `export`
5) `cat mempool-config.json`

Reply:
While writing this I realized that I accidentally ran `apps.restart.mutate` instead of `apps.update.mutate` 🤦 

Now that I updated the app, I can see that the lightning tab is working again and I see the correct exports.

Reply:
Thanks @knorrium, that all looks good. And so if you check the `env` of the `api` container right now does it show the correct variables or are they missing there too?

```
sudo docker exec mempool_api_1 env
```

Reply:
Here is mempool running on my Pi 5 right now:

```
$ sudo docker exec mempool_api_1 env | grep LND
LND_REST_API_URL=https://10.21.21.9:8080
LND_TIMEOUT=120000
LND_TLS_CERT_PATH=/lnd/tls.cert
LND_MACAROON_PATH=/lnd/data/chain/bitcoin/mainnet/readonly.macaroon
```

Reply:
@nmfretz just checked the env like that and I can confirm it's correct 🎉 

Thanks again!

Reply:
> While writing this I realized that I accidentally ran apps.restart.mutate instead of apps.update.mutate 🤦
>
> Now that I updated the app, I can see that the lightning tab is working again and I see the correct exports.

Whoops, sorry I missed this when responding. Haha, no worries at all. Glad it's sorted. Excited for the next release!